analytics_data_api/v0/serializers.py

```diff
@@ -329,35 +332,47 @@ class Meta(object):
         )
 
 
+# pylint: disable=abstract-method
 class LastUpdatedSerializer(serializers.Serializer):
-    last_updated = serializers.DateField(source='date', format=settings.DATE_FORMAT)
+    last_updated = serializers.DateTimeField(source='date', format=settings.DATE_FORMAT)
```

Comment:
Oh yeah!  I remember the problem I ran into with this initially.  I tried using a `DateField`, but elasticsearch stores everything as a datetime, even if it really is just a date.  When using a `DateField`, I get a "coercion" error. 😞